configs/dsdl/cityscapes.py

```diff
@@ -0,0 +1,70 @@
+_base_ = [
+    '../_base_/models/deeplabv3_r50-d8.py', '../_base_/default_runtime.py',
+    '../_base_/schedules/schedule_40k.py'
+]
+
+crop_size = (512, 1024)
+data_preprocessor = dict(size=crop_size)
+model = dict(data_preprocessor=data_preprocessor)
+# dataset settings
+dataset_type = 'DSDLSegDataset'
+data_root = 'data/CityScapes'
```

Comment:
```suggestion
data_root = 'data/cityscapes'
```

Reply:
I think this path should correspond to the one in the documentation.  https://mmsegmentation.readthedocs.io/en/latest/user_guides/2_dataset_prepare.html

Reply:
fixed